crates/sui-core/src/validator_client_monitor/monitor.rs

```diff
@@ -252,8 +255,8 @@ impl<A: Clone> ValidatorClientMonitor<A> {
     pub fn select_shuffled_preferred_validators(
```

Comment:
We probably need to refactor the comments as well to match the new approach (practically how we calculate `k` now)